OpenMarket/OpenMarket/Controller/RegisterViewController.swift

```diff
@@ -0,0 +1,170 @@
+//
+//  RegisterViewController.swift
+//  OpenMarket
+//
+//  Created by 우롱차, Donnie on 2022/05/25.
+//
+
+import UIKit
+
+final class RegisterViewController: RegisterEditBaseViewController {
+    
+    private enum Constant {
+        static let navigationTitle = "상품등록"
+    }
+    
+    private let picker = UIImagePickerController()
+    private let productListUseCase = ProductListUseCase()
+    
+    private lazy var addImageButton: UIButton = {
+        let imageButton = UIButton()
+        let image = UIImage(systemName: "plus")
+        imageButton.setImage(image, for: .normal)
+        imageButton.backgroundColor = .systemGray5
+        imageButton.addTarget(self, action: #selector(addImage), for: .touchUpInside)
+        return imageButton
+    }()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        picker.delegate = self
+        setBaseImage()
+        navigationItem.title = Constant.navigationTitle
+    }
+}
+
+// MARK: - Method
+extension RegisterViewController {
+    func addImageToStackView(image: UIImage){
+        let imageView = UIImageView(frame: CGRect(origin: .zero, size: CGSize(width: 50, height: 50)))
+        imageView.isUserInteractionEnabled = true
+        imageView.image = image
+        let gesture = UISwipeGestureRecognizer(target: self, action: #selector(deleteImageView))
+        gesture.direction = .up
+        imageView.addGestureRecognizer(gesture)
+        
+        addImageHorizontalStackView.addLastBehind(view: imageView)
+        
+        NSLayoutConstraint.activate([
+            imageView.heightAnchor.constraint(equalTo: addImageHorizontalStackView.heightAnchor),
+            imageView.widthAnchor.constraint(equalTo: imageView.heightAnchor)
+        ])
+        
+        if addImageHorizontalStackView.arrangedSubviews.count == 6 {
+            addImageButton.isHidden = true
+        } else {
+            addImageButton.isHidden = false
+        }
```

Comment:
좀 더 간단하게 바꿔볼 수 있을 것 같아요.
```suggestion
        addImageButton.isHidden = (addImageHorizontalStackView.arrangedSubviews.count == 6)
```
`(addImageHorizontalStackView.arrangedSubviews.count == 6)`를 따로 상수로 빼줘서 의미를 부여해주면 더 좋을 것 같구요!

Reply:
오... 요런 방법이... 참고해서 수정하겠습니다.

Reply:
또치의 조언대로 수정하였습니다!